andela/hc-millennium-falcon
# #156577924 Send alerts via multiple methods e.g SMS

### What does this PR do?
This PR enables an sms and telegram alert to be sent to a user to let them know about their check status. 

### Description of Task to be completed?
The user should be able to receive important information concerning their jobs through multiple channels like SMS and Telegram.

### How should this be manually tested?
1. Create a job with a timeout period of one minutes and grace period of one minute.
2.  Select the integrations tab and select SMS. 
3.  Follow instructions on how to integrate SMS
4.  Ping the job URL to activate it
5.  After that wait for the ping to go down
6.  An SMS alert should be sent to your phone number
7.  Do the same for telegram integration from step 2  to 5
8. A telegram text alert should be sent to your telegram application

### Any background context you want to provide?
Previously a user was not able to receive an alert about his/her check status using telegram and SMS. Now they can.

### What are the relevant pivotal tracker stories?
[#156577924](https://www.pivotaltracker.com/story/show/156577924)
